src/default_panels/GraphCreatePanel.js

```diff
@@ -30,8 +30,25 @@ const GraphCreatePanel = (props, {localize: _, setPanel}) => {
 
       <LocationSelector attr="type" />
 
-      <DataSelector label={_('Values')} attr="values" />
-      <DataSelector label={_('Labels')} attr="labels" />
+      <TraceTypeSection traceTypes={['pie']} mode="trace">
+        <DataSelector label={_('Values')} attr="values" />
```

Comment:
why isolate here instead of reuse?

Reply:
It seemed to me that the preferred order in which these show up in Pie vs Sunburst traces would be different..

I can remove if you don't mind the order

Reply:
I don't mind the order